### ISSUE TITLE
Speedment with TomEE

### ISSUE THREAD

**Issue by Rahul-khandelwal** [speedment/speedment#291]
Hi, I am using speedment with tomee and postgres db.
I have created a JDBC connection pool using tomcat resources.xml and speedment seems to work fine.
Is there any way I can confirm that sppedment is using connection from the pool I have created?

**Comment by minborg** [comment]
Speedment is using its own connection pool so there will be two layers of connection pooling in this case. It is difficult to see which connection implementation that is being used. Perhaps we could improve logging for this.

**Comment by minborg** [comment]
3.0.5 will introduce a new LogType

```
final HaresApplication app = new HaresApplicationBuilder()
   .withLogging(ApplicationBuilder.LogType.STREAM)
   .withLogging(ApplicationBuilder.LogType.PERSIST)
   .withLogging(ApplicationBuilder.LogType.UPDATE)
   .withLogging(ApplicationBuilder.LogType.REMOVE)
   .withLogging(ApplicationBuilder.LogType.CONNECTION) // NEW LOG TYPE in 3.0.5
   .withPassword("hare")
   .build();
```        


We can then get info on the connection types that are being handled internally and externally (e.g. TomEE connection pool)

<pre>
2017-03-06T04:28:20.018Z DEBUG [main] (#UPDATE) - UPDATE `hares`.`hare` SET `id` = ?,`name` = ?,`color` = ?,`age` = ? WHERE `id` = ?, values:[42, Harry, Gray, 10, 42]
2017-03-06T04:28:20.019Z DEBUG [main] (#CONNECTION) - getConnection(jdbc:mysql://127.0.0.1:3306/?useUnicode=true&characterEncoding=UTF-8&useServerPrepStmts=true&zeroDateTimeBehavior=convertToNull&useSSL=false&nullNamePatternMatchesAll=true&useLegacyDatetimeCode=true, hare, *****)
2017-03-06T04:28:20.030Z DEBUG [main] (#CONNECTION) - New external connection: com.mysql.jdbc.JDBC4Connection@6adede5
2017-03-06T04:28:20.030Z DEBUG [main] (#CONNECTION) - New Connection: com.speedment.runtime.core.internal.pool.PoolableConnectionImpl@2d928643
</pre>

**Comment by minborg** [comment]
Fixed in 214b9ba19f9cd43a201da702c6f4b1c6a875fc7d

**Comment by Pyknic** [comment]
Cool feature! I can definitely see usages for this.
